spec/item_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../item'
+
+describe Item do
```

Comment:
As per the project requirements, `move_to_archive` method in the item class should also be tested. Kindly follow-up on this requirement.